CODEOWNERS

```diff
@@ -51,4 +74,23 @@ platform/platform-paxweb-jettyconfig/*	@stustison
 
 platform/security/*		@stustison
 
+platform/security/security-migratable/*     @paouelle   @lambeaux
+
 platform/util/*			@stustison
+
+platform/migration/*            @paouelle   @lambeaux
+platform/platform-migratable/*  @paouelle   @lambeaux
+
+platform/platform-configuration/*       @lambeaux
+platform/platform-filter-clientinfo/*   @lambeaux
+
+
+###############################
+# ADMIN
+###############################
+
+platform/admin/*		@stustison
+
+platform/admin/core/*    @lambeaux
```

Comment:
I definitely thought about that. I saw benefits in terms of facilitating interests of the admin team but it still might be too much. And our team in particular has done a lot of research into config admin and related dependencies. I'll probably try to scope this down even more.